app/classify.py

```diff
@@ -1,37 +1,37 @@
-from datetime import datetime
+from astropy.coordinates import SkyCoord
+from datetime import datetime, timedelta
 from pathlib import Path
 
 import astropy.units as u
 import numpy as np
 from astropy.time import Time
-from sunpy.io.special import read_srs
+from sunpy.map import Map
+
 from sunpy.net import Fido, attrs as a
 
+CUTOUT =  [800, 400] * u.pix
+
 
-def download_srs(time: datetime):
+def download_magnetogram(time):
     r"""
-    Download SRS file for given time.
+    Download magnetogram and prepare for use
 
     Parameters
     ----------
-    time
-        Time to search for SRS file
-
-    Returns
-    -------
+    time : datetime.datetime
 
     """
-    time = Time(time)
-    tstart = time - 12 * u.hour
-    tend = time + 12 * u.hour
-    srs_query = Fido.search(a.Time(tstart, tend), a.Instrument.soon)
-    if 'srs' not in srs_query.keys():
-        raise SRSNotFoundError(f'No SRS file found for {time.isot}')
-    closest_idx = np.argmin(srs_query['srs']['Start Time'] - time)
-    srs_file = Fido.fetch(srs_query['srs'][closest_idx])
-    if len(srs_file.errors) != 0:
-        raise SRSDownloadError(f"Error downloading file {srs_query['srs']['url']}")
-    return srs_file
+    query = Fido.search(a.Time(time, time + timedelta(minutes=1)),
```

Comment:
Might need to switch to JSOC/DRMS as can get different series here e.g. 45s v 12minute but for the moment should be ok.